src/test/java/com/github/princesslana/eriscasper/data/TestData.java

```diff
@@ -66,6 +68,21 @@ public void toJsonString_whenEmptyListInMap_shouldInclude() throws DataException
         .isEqualTo("{\"key\":[]}");
   }
 
+  @Test
+  public void toQueryString_whenPopulated_shouldFormat() {
+    Assertions.assertThat(Data.toQueryString(Obj.of(Optional.of("test")))).isEqualTo("field=test");
```

Comment:
Do we need a test for Optional.empty?

Reply:
Right now Optional.empty() produces null for some reason. I'm not exactly sure why since we have the filter which works on other tests and it's using the same principle.

Reply:
Nevermind, I'm wrong, it works, I'll add the test.